ravedude/src/main.rs

```diff
@@ -341,7 +351,9 @@ fn ravedude() -> anyhow::Result<()> {
                 "-b/--baudrate is needed for the serial console"
             })?;
 
-        let port = port.context("console can only be opened for devices with USB-to-Serial")?;
+        let port = console_port
+            .or_else(|| port)
+            .context("console can only be opened for devices with USB-to-Serial")?;
```

Comment:
This error message is no longer accurate.  I would suggest the following:

```suggestion
            .context("no programmer serial port or `console-port` was set")
            .context("cannot open console without a serial port")?;
```